lib/csv_helper/exporter.rb

```diff
@@ -18,10 +18,17 @@ def generate(csv_headers)
       CSV.generate do |csv|
         csv << csv_headers.values
 
-        klass.find_each do |record|
+        set_scope.find_each do |record|
           csv << csv_headers.keys.map { |k| record[k] }
         end
       end
     end
+
+    def set_scope
```

Comment:
is scope the best name to use? It does make sense, but might be confusing seeing as scope has semantical meaning in Rails.